lib/export-control.ts

```diff
@@ -16,7 +16,7 @@ type Options = {
   Crosshair?: boolean;
   PrintableArea: boolean;
   accessToken?: string;
-  Local?: 'de' | 'en' | 'fr' | 'fi' | 'sv' | 'vi';
+  Local?: 'de' | 'en' | 'fr' | 'fi' | 'sv' | 'vi' | 'zh_Hans_CN' | 'zh_Hant_CN';
```

Comment:
As I commented previously, we can use `-` and we may not need regional name like `CN` in a language code.